discord_typings/resources/voice.py

```diff
@@ -41,3 +41,13 @@ class VoiceRegionData(TypedDict):
     optimal: bool
     deprecated: bool
     custom: bool
+        
+        
+# https://discord.com/developers/docs/topics/gateway#voice-server-update-voice-server-update-event-fields
+
+
+@final
+class VoiceServerData(TypedDict)
```

Comment:
You're missing a colon:

Reply:
Yes this is what appears to be causing the lint failure.